torchrl/envs/common.py

```diff
@@ -1252,8 +1249,12 @@ def policy(td):
                 break
             tensordict = step_mdp(
                 tensordict,
-                keep_other=True,
-                exclude_action=False,
+                keep_other=False,
+                exclude_action=True,
```

Comment:
If we set `keep_other=False`,`exclude_action=True` we may save time.
Was there a reason they were set differently? I might be missing something